docs/modules/player-lifecycle/basic-player-creation.md

```diff
@@ -0,0 +1,94 @@
+
+Create a method that returns an `EntityTemplate` object and takes the following parameters to define your player [entity template]({{urlRoot}}/reference/concepts/entity-templates):
+
+* `string workerId`: The ID of the worker that wants to spawn this player entity.
+*  `byte[] playerCreationArguments`: a serialized byte array of arguments provided by the worker instance sending the player creation request.
+
+When defining the entity template, you need to use the `AddPlayerLifecycleComponents` method. This method adds lifecycle SpatialOS components to the player entity template; these components are necessary to manage the player lifecycle for this entity.
+
+The following code snippet shows an example on how to implement such a method:
+
+```csharp
+public static class PlayerTemplate
+{
+    public static EntityTemplate CreatePlayerEntityTemplate(string workerId, byte[] playerCreationArguments)
+    {
+        // Obtain unique client attribute of the client-worker that requested the player entity
+        var clientAttribute = EntityTemplate.GetWorkerAccessAttribute(workerId);
+        // Obtain the attribute of your server-worker
+        var serverAttribute = "UnityGameLogic";
+
+        var entityTemplate = new EntityTemplate();
+        entityTemplate.AddPosition(new Position.Snapshot(new Coordinates()), serverAttribute);
+        // add all components that you want the player entity to have
+        AddPlayerLifecycleComponents(entityTemplate, workerId, serverAttribute);
+
+        return entityTemplate;
+    }
+}
+```
+
+## Configure an entity template to use for player creation
+
+You need to configure the `PlayerLifecycleConfig` configuration class to use the player entity template. Do this by setting the `PlayerLifecycleConfig.CreatePlayerEntityTemplate` field to `PlayerTemplate.CreatePlayerEntityTemplate`.
+
+The following example is a code snippet using a script that runs once when starting the game to initialize the configuration for the player lifecycle module:
+
+```csharp
+public static class OneTimeInitialization
+{
+    private static bool initialized;
+
+    // Using this attribute will trigger the Init() method whenever a Scene gets loaded.
+    [RuntimeInitializeOnLoadMethod]
+    private static void Init()
+    {
+        if (initialized)
+        {
+            // ensures to only run it the first time a Scene gets loaded.
+            return;
+        }
+
+        initialized = true;
+        // initialize your game...
+        // ...
+        // Setup template to use for player on connecting client
+        PlayerLifecycleConfig.CreatePlayerEntityTemplate = PlayerTemplate.CreatePlayerEntityTemplate;
+    }
+}
+```
+
+## When is a player entity deleted?
```

Comment:
This feels out of place here.

We can point users to the heartbeating _or_ custom player creation docs but probably not worth talking in too much detail